nservicebus/licensing/license-management.md

```diff
@@ -110,3 +110,22 @@ As a developer you can specify the license to use in your configuration code:
 
 <!-- import License -->
 
+## Where does NServiceBus look for a license?
```

Comment:
so the above content references different version behaviour.

Is the below content correct for all versions of NSB?